### PR TITLE
[Improve] add OTG status color

### PR DESCRIPTION
**Pull request by jaluowen**
-Add OTG status color

### CODE REVIEW

**Comment by jaluowen** [comment]
Review kang @gaganugraha33 @anandarh @aldiiii

File: lib/screens/myAccount/ProfileScreen.dart [hunk 1]
```diff
@@ -515,6 +515,12 @@ class _ProfileScreenState extends State<ProfileScreen> {
               : Colors.red);
           break;
 
+        case "OTG":
```
Comment (gaganugraha33) [inline lib/screens/myAccount/ProfileScreen.dart:518]: text casenya di simpen ke dictionary aja gans

Comment (jaluowen) [reply lib/screens/myAccount/ProfileScreen.dart:518]: oiyaaa siaap

Comment (jaluowen) [reply lib/screens/myAccount/ProfileScreen.dart:518]: eh gabisa kang "Case expressions must be constant" katanya

Comment (gaganugraha33) [reply lib/screens/myAccount/ProfileScreen.dart:518]: eh iya ya wkwk kirain bisa 😅 @jaluowen , di close dulu

**Pull request merged**